Sources_v3/StreamChat/WebSocketClient/WebSocketClient.swift

```diff
@@ -92,22 +88,24 @@ class WebSocketClient {
         return pingController
     }()
     
-    private func createEngine() -> WebSocketEngine {
+    private func createOrReuseEngine(for connectEndpoint: Endpoint<EmptyResponse>) -> WebSocketEngine {
```

Comment:
`createEngineIfNeeded`? 🤯

Reply:
fixed in c732d658ca9625bd74d35c32c4d80b3a70871a0e